$NetBSD$

- Also depends on openjpeg, jpeg and jbig2dec to avoid missing PLT symbols at
  run time. Please note that this was addressed upstream via:

   <https://git.pwmt.org/pwmt/zathura-pdf-mupdf/commit/8d27748c525691268305c6760c7d4668d0560313>

- Adjust library name of mupdf-third for mupdf>=1.14.0.

--- meson.build.orig	2018-03-17 19:47:01.000000000 +0000
+++ meson.build
@@ -19,10 +19,13 @@ zathura = dependency('zathura', version:
 girara = dependency('girara-gtk3')
 glib = dependency('glib-2.0')
 cairo = dependency('cairo')
+libopenjp2 = dependency('libopenjp2')
+libjpeg = dependency('libjpeg')
+jbig2dec = cc.find_library('jbig2dec')
 mupdf = cc.find_library('mupdf')
-mupdfthird = cc.find_library('mupdfthird')
+mupdfthird = cc.find_library('mupdf-third')
 
-build_dependencies = [zathura, girara, glib, cairo, mupdf, mupdfthird]
+build_dependencies = [zathura, girara, glib, cairo, libopenjp2, libjpeg, jbig2dec, mupdf, mupdfthird]
 
 # defines
 defines = [
